go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/iver-wharf/wharf-api-client-go/v2
 
-go 1.13
+go 1.16
 
 require (
-	github.com/iver-wharf/wharf-core v1.1.0
+	github.com/google/go-querystring v1.1.0
+	github.com/iver-wharf/wharf-api/v5 v5.0.0
```

Comment:
I'm torn about depending on `wharf-api`.. Duplicating the models ought to be simple enough now that they're in a separate package.

Reason being that even though we're not using all of wharf-api's dependencies, we now still force all dependants of wharf-api-client-go to also depend on Sqlite3, GORM, Gin, etc. While this does not affect the size of the built binaries, it does affect the build time as `go get` and `go mod download` will download all dependencies used by any package, no matter if they're referenced in the code or not.

I suggest duplicating the `wharf-api/pkg/model/{request,response}` directories into `wharf-api-client-go/pkg/model/{request,response}`.

We could potentionally use `go generate` to automate this task in the future, if need be.

Reply:
Having the option of using `go generate` later makes me more comfortable with not having a direct link to them, as then it doesn't have to be done manually each time it changes, which is the only thing I really wanted to avoid. 👍🏻 

For now copying over request/response in its entirety, with a comment saying `Copied from github.com/iver-wharf/wharf-api/pkg/model/{folder}` in both of the files `request.go` `response.go`.